scram/users/tests/factories.py

```diff
@@ -1,6 +1,7 @@
 """Define Factory tests for the Users application."""
 
-from typing import Any, Sequence
+from collections.abc import Sequence
```

Comment:
Is there a difference in the sequence modules?

Reply:
Apparently one is deprecated:

> if you are on Python 3.9+, you should most likely never use [typing.Sequence](https://docs.python.org/3/library/typing.html#typing.Sequence) due to its deprecation. Since the introduction of [generic alias types](https://docs.python.org/3/library/stdtypes.html#types-genericalias) in 3.9 the [collections.abc](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes) classes all support subscripting and should be recognized correctly by static type checkers of all flavors.